fix grammar & typos (#4306)
fastapi/fastapi: docs/en/docs/deployment/concepts.md

### Restart After Crash

> But in those cases with awful errors that crash the running **process**, you would want an external component that is in charge of **restarting** the process, at least a couple of times...

Reviewer: ```suggestion
But, in cases of a hard error, that crashes the running **process**, you would want an external component that is in charge of **restarting** the process, at least a couple of times...
```

## Recap

> You have been reading here some main concepts that you would probably need to have in mind when deciding how to deploy your application:

Reviewer: ```suggestion
You have been reading here the main concepts that you would probably need to have in mind when deciding how to deploy your application:
```
